lib/success.js

```diff
@@ -5,7 +5,7 @@ const template = require('./template')
 const truncate = require('./truncate')
 const getRepoInfo = require('./getRepoInfo')
 
-// Max lenght of message to the slack api is 3000 characters. Make sure we are below that.
+// Max length of message to the slack api is 40,000 characters. 2900 is based on the old limit.
```

Comment:
I think the old comment is wrong, as the limit is specific to `section`. The new comment does not really explain why `MAX_LENGTH ` is 2900 though. Do you think we should change the last sentence to something that explains why 2900 is chosen?

Like `2900 is the limit for a message block of type 'section'.`

Reply:
Done